src/fragments/lib-v1/auth/js/user_attributes/update_user_attributes.mdx

```diff
@@ -0,0 +1,90 @@
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { updateUserAttributes, UpdateUserAttributesOutput } from 'aws-amplify/auth';
+
+async function handleUpdateEmailAndNameAttributes(
+	updatedEmail: string,
+	updatedName: string
+) {
+	try {
+		const attributes = await updateUserAttributes({
+			userAttributes: {
+				email: updatedEmail,
+				name: updatedName,
+			},
+		});
+		handleUpdateUserAttributesOutput(attributes);
+	} catch (error) {
+		console.log(error);
+	}
+}
+
+function handleUpdateUserAttributesOutput(
+	attributes: UpdateUserAttributesOutput
+) {
+	Object.entries(attributes).forEach(([attribute, output]) => {
+		const { nextStep } = output;
+		switch (nextStep.updateAttributeStep) {
+			case 'CONFIRM_ATTRIBUTE_WITH_CODE':
+				const codeDeliveryDetails = nextStep.codeDeliveryDetails;
+				console.log(
+					`Confirmation code was sent to ${codeDeliveryDetails?.deliveryMedium}.`
+				);
+				const confirmationCode = '112233' // code retrieved from phone or email.
+				handleConfirmUserAttribute(attribute, confirmationCode);
+				break;
+			case 'DONE':
+				console.log(`${attribute} attribute was successfully updated.`);
+				break;
+		}
+	});
+}
+```
+</Block>
+<Block name="JavaScript">
+
+```javascript
+import { updateUserAttributes } from 'aws-amplify/auth';
+
+async function handleUpdateEmailAndNameAttributes(
+	updatedEmail,
+	updatedName
+) {
+	try {
+		const attributes = await updateUserAttributes({
+			userAttributes: {
+				email: updatedEmail,
+				name: updatedName,
+			},
+		});
+		handleUpdateUserAttributesOutput(attributes);
+	} catch (error) {
+		console.log(error);
+	}
+}
+
+function handleUpdateUserAttributesOutput(
+	attributes
+) {
+	Object.entries(attributes).forEach(([attribute, output]) => {
+		const { nextStep } = output;
+		switch (nextStep.updateAttributeStep) {
+			case 'CONFIRM_ATTRIBUTE_WITH_CODE':
+				const codeDeliveryDetails = nextStep.codeDeliveryDetails;
+				console.log(
+					`Confirmation code was sent to ${codeDeliveryDetails?.deliveryMedium}.`
+				);
+				const confirmationCode = '112233' // code retrieved from phone or email.
+				handleConfirmUserAttribute(attribute, confirmationCode);
+				break;
```

Comment:
For some reason the syntax highlighting looks off in Github for me starting here. Is it okay in your doc site preview? just as a sanity check

Reply:
it was off for me too. I updated to the right syntax. Thanks